dask_cuda/tests/test_device_host_file.py

```diff
@@ -14,6 +15,13 @@
 cupy = pytest.importorskip("cupy")
 
 
+def test_device_host_file_config(tmp_path):
+    dhf_disk_path = str(tmp_path / "dask-worker-space" / "storage")
+    with dask.config.set(temporary_directory=str(tmp_path)):
+        dhf = DeviceHostFile()
+        assert dhf.disk_func_path == dhf_disk_path
```

Comment:
Wouldn't it make more sense to check if the directory indeed got created? We could also avoid the need to make `disk_func_path` an attribute of `DeviceHostFile`.

Reply:
That seems like a good check to add. Happy to add that.

That said, it still seems worthwhile to verify `DeviceHostFile` is using that path.